src/embed/embedConfig.ts

```diff
@@ -4,8 +4,49 @@ import { EmbedConfig } from '../types';
 const configKey = 'embedConfig';
 
 /**
- * Gets the configuration embed was initialized with.
- * @returns {@link EmbedConfig} The configuration embed was initialized with.
+ * Gets the `EmbedConfig` object that was used to
+ * initialize the SDK. You can use this method to access the
+ * embed configuration configuration settings such as the ThoughtSpot host,
+ * authentication type, and other such parameters used when
+ * initializing the SDK.
+ *
+ * @example
+ * ```
+ * const config = getInitConfig();
```

Comment:
I see this as description for `getEmbedConfig` instead of `getInitConfig`?

Reply:
The SDK docs point to this section (embedConfig.ts#L12) - https://developers.thoughtspot.com/docs/Function_getInitConfig.